gallery/src/styles/NxCard/NxCardPage.tsx

```diff
@@ -0,0 +1,152 @@
+              replacing this style with <code className="nx-code">.nx-card-container--row</code> above.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              Basic card element.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card-header</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              Card header element, typically wraps an <code className="nx-code">&lt;h3&gt;</code>.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__call-out</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              Wraps the card call out (if any). The call out typically consists of a number, sparkline, icon, or
+              other small graphic. It should not be free form text.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-call__call-out-text</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              In the case where the call out consists of text (e.g. 99%) it should be wrapped in this class (typically
+              in a <code className="nx-code">&lt;span&gt;</code>).
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__text</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">Free-form text. Should be concise.</td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card-footer</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">Card footer. Typically provides a link to more information.</td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card--vertical-alt</code></td>
+            <td className="nx-cell">Modifier of <code className="nx-code">.nx-card</code></td>
+            <td className="nx-cell">
+              In order to use the variant vertical layout this class should be applied to
+              {' '}<code className="nx-code">.nx-card</code> in conjunction with a
+              {' '}<code className="nx-code">&lt;div&gt;</code> wrapping the call out and description text.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card-content</code></td>
```

Comment:
So this cannot be used in a card that may covert to horizontal layout, correct?  Would I be correct in thinking that if `display: contents` worked in all supported browsers, we could avoid that restriction?

Reply:
> So this cannot be used in a card that may covert to horizontal layout, correct? Would I be correct in thinking that if `display: contents` worked in all supported browsers, we could avoid that restriction?

I didn't play with `display: contents` enough to give a very definitive answer, but I believe so yes. If it weren't for IE I would have explored this further.